README.md

```diff
@@ -23,63 +33,55 @@ More details...
 -
 
 ## Usage
-To run:
-```sh
-$ reponame dostuff arg1 arg2
-```
 ```js
 reponame.doStuff(arg1, arg2)
 ```
 
-## Example
-```sh
-$ reponame dostuff 32 'ara-docs'
-*EXAMPLE RESPONSE HERE*
-```
-
+### Example
 ```js
 const reponame = require('ara-reponame')
 
 const result = reponame.doStuff(32, 'ara-docs')
-console.log(result)
+`${console.log(result)}`
```

Comment:
Typo?

Reply:
so with this... I'm unsure of how to communicate that the _actual_ logged out result should be here. (not _literally_ `console.log(result)`... thoughts?